docs/platforms/go/guides/logrus/index.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Logrus
+description: "Logrus is a structured logger for Go, used to log messages in different formats and levels. This guide demonstrates how to integrate Logrus with Sentry to capture and send logs to Sentry."
+---
+
+For a quick reference, there is a [complete example](https://github.com/getsentry/sentry-go/tree/master/_examples/logrus) at the Go SDK source code repository.
+
+[Go Dev-style API documentation](https://pkg.go.dev/github.com/getsentry/sentry-go/logrus) is also available.
```

Comment:
```suggestion
Go API documentation for the [`sentrylogrus` package](https://pkg.go.dev/github.com/getsentry/sentry-go/logrus) is also available.
```
Original link text was vague and sounded less relevant. I think something like this would make me more likely to click the link. Feel free to adjust the text.